srcopsmetrics/metrics.py

```diff
@@ -121,14 +127,37 @@ def plot_graph_for_metrics(self, metrics_name: str, time_metrics_name: str):
         )
         plt.legend(loc="upper left")
 
-        path = f"./vismetrics_name/knowledge_statistics/{self.repo_name}"
+        path = f"./srcopsmetrics/knowledge_statistics/{self.repo_name}"
         check_directory(Path(path))
         plt.savefig(f"{path}/{metrics_name}", pad_inches=5)
         _LOGGER.info("Saved visualization %s", path)
         plt.clf()
 
-    def get_metrics_for_prs(self):
-        """Get metrics from Pull Requests."""
+    def compute_predictions(self, time_metrics_name: str, days_ahead: int = 7) -> np.array:
+        """Compute estimation of the mean metrics in time for future score.
+
+        Return numpy.array with prediciton for all the available dates
+        in self.pr_metrics plus specified days_ahead
+        """
+        score = np.poly1d(np.polyfit(self.pr_metrics["date"], self.pr_metrics[time_metrics_name], 3))
+        return score(
+            self.pr_metrics["date"].append(pd.Series([int(time.time()) * 3600 * 24 for i in range(1, days_ahead + 1)]))
+        )
+        # one_week_ahead_timestamp = int(time.time()) + 3600*24*7
```

Comment:
do we keep commented code?

Reply:
you're looking at outdated version :stuck_out_tongue_winking_eye:

Reply:
after changes yes :P